UIKit/Tasks/Picker/GenderPicker.swift

```diff
@@ -0,0 +1,47 @@
+// GenderPicker.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Класс где создан пикер для выбора пола
+class GenderPicker: UIPickerView {
+    var people: [String] = ["Male", "Female"]
```

Comment:
некорректное имя переменной + это неизменяемая переменная, + надо заприватить
```suggestion
    private let genders: [String] = ["Male", "Female"]
```